src/lib/meta.js

```diff
@@ -52,3 +54,39 @@ module.exports.polyline = function(context) {
         alert('Sorry, we were unable to decode that polyline');
     }
 };
+
+module.exports.wkxBase64 = function(context) {
+    var input = prompt('Enter your Base64 encoded WKB/EWKB');
+    try {
+        var decoded = wkx.Geometry.parse(Buffer.from(input,'base64'));
+        context.data.set({ map: decoded.toGeoJSON() });
+        zoomextent(context);
+    } catch(e) {
+        console.error(e)
+        alert('Sorry, we were unable to decode that Base64 encoded WKX data');
+    }
+};
+
+module.exports.wkxHex = function(context) {
+    var input = prompt('Enter your Hex encoded WKB/EWKB');
+    try {
+        var decoded = wkx.Geometry.parse(Buffer.from(input,'hex'));
+        context.data.set({ map: decoded.toGeoJSON() });
+        zoomextent(context);
+    } catch(e) {
+        console.error(e)
+        alert('Sorry, we were unable to decode that Base64 encoded WKX data');
```

Comment:
Think you put your comment in the wrong place.  This block is base64; hex is a different block, and did need fixed.  it now is.